includes/admin/class-wc-stripe-settings-controller.php

```diff
@@ -10,7 +10,15 @@
  * @since 5.4.1
  */
 class WC_Stripe_Settings_Controller {
+	/**
+	 * Stripe Account
+	 *
+	 * @var WC_Stripe_Account
+	 */
+	private $account;
+
 	public function __construct() {
+		$this->account = new WC_Stripe_Account();
```

Comment:
What do you think about injecting this in the constructor, rather than instantiating it here?
That way you can more easily mock it for the tests